softkinetic_camera/src/softkinetic_start.cpp

```diff
@@ -500,6 +495,14 @@ void onNewDepthSample(DepthNode node, DepthNode::NewSampleReceivedData data)
       current_cloud->points[count].g = cv_img_rgb.at<cv::Vec3b>(y_pos, x_pos)[1];
       current_cloud->points[count].r = cv_img_rgb.at<cv::Vec3b>(y_pos, x_pos)[2];
     }
+    else
+    {
+      continue;
+    }
+    // Convert softkinetic vertices into a kinect-like coordinates pointcloud
+    current_cloud->points[count].x =   data.verticesFloatingPoint[count].x + 0.025;
```

Comment:
this is offset due to softkinetic lense position
i got this value from code below . 
https://github.com/ipa320/softkinetic/blob/indigo_dev/softkinetic_camera/urdf/senz3d.urdf.xacro#L38-L55
i think it's correct and it only has offset in x.